common/main/java/com/couchbase/lite/Collection.java

```diff
@@ -46,7 +46,36 @@
 
 
 /**
+ * A <code>Collection</code> is a container for documents similar to aa table in a relational
```

Comment:
This is a backport of the documentation on the Collections class.  Not really related to this PR